tools/include_check.sh

```diff
@@ -1,15 +1,22 @@
-#!/bin/sh
+#!/bin/sh -u
 
 # Copyright (c) 2019 Yubico AB. All rights reserved.
 # Use of this source code is governed by a BSD-style
 # license that can be found in the LICENSE file.
 # SPDX-License-Identifier: BSD-2-Clause
 
+SKIP='(webauthn.h)'
+
 check() {
-	for f in $(find $1 -maxdepth 1 -name '*.h'); do
-		echo "#include \"$f\"" | \
-			cc $CFLAGS -Isrc -xc -c - -o /dev/null 2>&1
-		echo "$f $CFLAGS $?"
+	try="cc $CFLAGS -Isrc -xc -c - -o /dev/null 2>&1"
+	git ls-files "$1" | grep '.*\.h$' | while read -r header; do
+		if echo "$header" | grep -Eq "$SKIP"; then
+			echo "Skipping $header"
+		else
+			body="#include \"$header\""
+			echo "echo $body | $try"
+			echo "#include \"$header\"" | eval "$try"
```

Comment:
Since we have it, would `echo $body` make sense here?

Reply:
Sure; addressed in new branch